### ISSUE TITLE
split up vgrid-generator

### ISSUE THREAD

**Issue by Thanood** [vegarringdal/aureliaGrid#41]
I find the generator code hard to follow.
Is it possible to split the html generation, scroll handlers and (click) event handlers into seperate files?

**Comment by vegarringdal** [comment]
Ill start with the scrollhandlers today.
Some communication is a bit messy between the classes I have, splitting the code might help out.
Cleared a lot of that generator code yesterday, but added a lot that was missing, like unbind/detatch of the viewports of the new one.

**Comment by vegarringdal** [comment]
Started on it, now the scroll functions are in own class, and removed a lot of junk code in the generator.
Have more to do, but needed to update with what I had since it was working, and I will prb break something on my next project

**Comment by vegarringdal** [comment]
Made more improvements to this class today, moved more functions out, and reordered them so main functions are more grouped for creating viewslots and elements
I can prb do more, but have other things that needs more attention
Closingthis for now, juts create a new issue if you see something. Just updated the code
